src/Common/src/Interop/Unix/System.Native/Interop.MountPoints.FormatInfo.cs

```diff
@@ -143,91 +191,154 @@ private static DriveType GetDriveType(string fileSystemName)
+                case "cgroupfs":
+                case "coh":
+                case "cramfs":
+                case "cramfs-wend":
+                case "daxfs":
                 case "drvfs":
                 case "ecryptfs":
+                case "efivarfs":
                 case "efs":
+                case "exofs":
                 case "ext":
                 case "ext2":
                 case "ext2_old":
                 case "ext3":
+                case "ext2/ext3":
                 case "ext4":
                 case "ext4dev":
+                case "f2fs":
                 case "fat":
-                case "fuseblk":
                 case "fuseext2":
                 case "fusefat":
+                case "futexfs":
                 case "hfs":
+                case "hfs+":
                 case "hfsplus":
+                case "hfsx":
+                case "hostfs":
                 case "hpfs":
+                case "inodefs":
+                case "inotifyfs":
                 case "jbd":
                 case "jbd2":
-                case "jfs":
                 case "jffs":
                 case "jffs2":
+                case "jfs":
+                case "logfs":
                 case "lxfs":
+                case "minix (30 char.)":
+                case "minix v2 (30 char.)":
+                case "minix v2":
                 case "minix":
                 case "minix_old":
                 case "minix2":
                 case "minix2v2":
+                case "minix3":
+                case "mlfs":
                 case "msdos":
+                case "nilfs":
+                case "nsfs":
+                case "ntfs":
                 case "ocfs2":
                 case "omfs":
                 case "openprom":
                 case "overlay":
                 case "overlayfs":
-                case "ntfs":
+                case "pstorefs":
                 case "qnx4":
+                case "qnx6":
                 case "reiserfs":
+                case "rpc_pipefs":
+                case "selinux":
+                case "smackfs":
                 case "squashfs":
                 case "swap":
+                case "sysfs":
                 case "sysv":
+                case "sysv2":
+                case "sysv4":
+                case "tracefs":
                 case "ubifs":
                 case "udf":
                 case "ufs":
                 case "umsdos":
                 case "umview-mod-umfuseext2":
+                case "usbdevfs":
+                case "v9fs":
+                case "vzfs":
+                case "wslfs":
+                case "xenfs":
                 case "xenix":
                 case "xfs":
+                case "xia":
                 case "xiafs":
                 case "xmount":
+                case "zfs":
                 case "zfs-fuse":
+                case "zsmallocfs":
                     return DriveType.Fixed;
 
                 case "9p":
-                case "autofs":
```

Comment:
autofs isn't necessarily for the network... it could be for any auto-mounted file system, e.g. for removable media.